javalib/src/main/scala/java/io/StringReader.scala

```diff
@@ -23,6 +27,9 @@ class StringReader(s: String) extends Reader {
   }
 
   override def mark(readAheadLimit: Int): Unit = {
+    if (readAheadLimit < 0) {
+      throw new IllegalArgumentException("Read-ahead limit < 0")
+    }
```

Comment:
No need for braces since there is only one statement in the `if` branch. See https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#braces